### PR TITLE
Adding WIP compendium to repository

### PR DESCRIPTION
**Pull request by teqqqie**
A compendium of creatures intended to add flavor, plot hooks, and non-combat challenges. Also includes a legendary group. My first attempt adding to the repository, so apologies for any mistakes.

### CODE REVIEW

File: creature/teqqqie; Weird Flavor Creatures.json [hunk 1]
```diff
@@ -0,0 +1,5636 @@
+{
+	"siteVersion": "1.149.1",
```
Comment (revilowaldow) [inline creature/teqqqie; Weird Flavor Creatures.json:2]:
```suggestion

```

File: creature/teqqqie; Weird Flavor Creatures.json [hunk 2]
```diff
@@ -0,0 +1,5636 @@
...
+	"_meta": {
+		"sources": [
+			{
+				"json": "WeirdFlavorCreatures",
+				"abbreviation": "WFC",
+				"full": "Weird Flavor Creatures",
+				"url": "https://homebrewery.naturalcrit.com/share/-s4yCQ_RQSi0",
```
Comment (revilowaldow) [inline creature/teqqqie; Weird Flavor Creatures.json:9]:
```suggestion
				"url": "https://homebrewery.naturalcrit.com/share/-s4yCQ_RQSi0",
				"version": "2021-05-27T03:51:20.571Z",
```

**Review by revilowaldow — CHANGES_REQUESTED** [review]

File: creature/teqqqie; Weird Flavor Creatures.json [hunk 3]
```diff
@@ -0,0 +1,5636 @@
...
+				"L",
+				"N",
+				"P"
+			],
+			"miscTags": [
+				"AOE",
+				"MW",
+				"RW"
+			]
+		},
```
Comment (TheGiddyLimit) [inline creature/teqqqie; Weird Flavor Creatures.json:1466]:
```suggestion
```

Comment (TheGiddyLimit) [inline creature/teqqqie; Weird Flavor Creatures.json:1466]:
```suggestion
```

Comment (teqqqie) [reply creature/teqqqie; Weird Flavor Creatures.json:1466]: Sorry, I'm new to github, what is changing here?

Comment (TheGiddyLimit) [reply creature/teqqqie; Weird Flavor Creatures.json:1466]: if you check the `Checks` tab at the top of the PR, you can see [what it's failing on](https://github.com/TheGiddyLimit/homebrew/pull/1262/checks)

which is, you've got two copies of the exact same monster; this change is to delete one of those copies

**Pull request merged**